_posts/en/newsletters/2023-12-13-newsletter.md

```diff
@@ -0,0 +1,202 @@
+  block data][stale block github].
+
+- **Envoy 1.4 released:**
+  Bitcoin wallet Envoy’s [1.4 release][envoy v1.4.0] adds [coin control][topic
+  coin selection] and [wallet labeling][topic wallet labels] ([BIP329][]
+  coming soon), among other features.
+
+- **BBQr encoding scheme announced:**
+  The [scheme][bbqr github] can efficiently encode larger files, for example [PSBTs][topic
+  psbt], into an animated QR series for use in air-gapped wallet configurations.
+
+- **Zeus v0.8.0 released:**
+  The [v0.8.0][zeus v0.8.0] release contains an embedded LND node, additional
+  [zero conf channel][topic zero-conf channels] support, and support for simple taproot channels,
+  among other changes.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aa
+nswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What are all the rules related to CPFP fee bumping?]({{bse}}120853)
+  Pieter Wuille points out that contrary to the [RBF][topic rbf] fee bumping
+  technique that has a list of associated policy rules, the [CPFP][topic cpfp]
+  fee bumping technique has no additional policy rules.
+
+- [How is the total number of RBF replaced transactions calculated?]({{bse}}120823)
+  Murch and Pieter Wuille walk through some examples of RBF replacements in the
+  context of [BIP125][]’s rule 5: “The number of original transactions to be
+  replaced and their descendant transactions which will be evicted from the
+  mempool must not exceed a total of 100 transactions”. Readers may also be
+  interested in the [Add BIP-125 rule 5 testcase with default
+  mempool][review club 25228] PR Review Club meeting.
+
+- [What types of RBF exist and which one does Bitcoin Core support and use by default?]({{bse}}120749)
+  Murch provides some of Bitcoin Core’s transaction replacement history and in a
+  [related question]({{bse}}120773), a summary of RBF replacement rules and
+  links to Bitcoin Core’s [Mempool Replacements][bitcoin core mempool
+  replacements] documentation and one developer's ideas for [RBF
+  improvements][glozow rbf improvements].
+
+- [What is the Block 1,983,702 Problem?]({{bse}}120834)
+  Antoine Poinsot gives an overview of the issues that led to [BIP30][]
+  restricting duplicate txids and [BIP34][] mandating the inclusion of the
+  current block height in the coinbase field. He then points out that there are
+  numerous blocks whose random coinbase field content happens to match the
+  mandatory height prefix of a later block. Block 1 983 702 being the first
```

Comment:
```suggestion
  mandatory height prefix of a later block. Block 1 983 702 is the first
```
Also, I don't think I've see spaces within number groups like this, maybe should be commas?